ibm/resource_ibm_is_lb_listener.go

```diff
@@ -309,6 +346,8 @@ func lbListenerGet(d *schema.ResourceData, meta interface{}, lbID, lbListenerID
 	}
 	d.Set(isLBListenerLBID, lbID)
 	d.Set(isLBListenerPort, *lbListener.Port)
+	d.Set(isLBListenerPortMin, *lbListener.PortMin)
```

Comment:
Do we need to check here lbListener.PortMin is not nil before assigning?  same with the Portmax and Port